lib/pbench/cli/server/options.py

```diff
@@ -22,13 +22,15 @@ def _pbench_server_config(f: Callable) -> Callable:
 
     def callback(ctx, param, value):
         clictx = ctx.ensure_object(CliContext)
-        clictx.config = value
+        clictx.config = (
+            value if value else "/opt/pbench-server/lib/config/pbench-server.cfg"
+        )
```

Comment:
> This works in all four cases?
> 
> * The user specifies the command line option with a valid path

Yes

> * The user omits the command line option but the environment variable is defined to a valid path

Yes

> * The user omits the command line option and the environment variable is undefined

That's the whole point, and, yes

> * The command line option, if provided, and the environment variable, if defined, point to an empty string

Actually, don't know and don't really care, but maybe I'll check. 😏 

> (I think it probably does, but I thought I'd confirm.)

Reply:
OK, intriguing:

`-C ""` fails because `Path '' does not exist.`: Click is explicitly looking for a `Path` that exists and is readable, and `""` isn't.

With `required=True`, `_PBENCH_SERVER_CONFIG= ` also fails, because the option is in fact required, but with `required=False` we take the default assignment and all is OK.

(With a valid path, the environment variable goes through Click's `Path` resolution and we don't require it to be unrequired...)

Reply:
Thanks for checking!